applications/rag/variables.tf

```diff
@@ -273,7 +273,7 @@ variable "cloudsql_instance" {
 variable "cloudsql_instance_region" {
   type        = string
   description = "GCP region for CloudSQL instance"
-  default     = "us-central1"
+  default     = ""
```

Comment:
either delete this line and force the user to specify a region, or leave a meaningful default